tomcat/src/main/java/org/apache/coyote/http11/RequestMapping.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.coyote.http11;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.coyote.http11.controller.Controller;
+import org.apache.coyote.http11.controller.LoginController;
+import org.apache.coyote.http11.controller.PageController;
+import org.apache.coyote.http11.controller.RegisterController;
+import org.apache.coyote.http11.controller.RootController;
+
+public class RequestMapping {
+
+    private static final Map<String, Controller> REQUEST_MAPPER = new HashMap<>();
+
```

Comment:
```suggestion
    private static final Map<String, Controller> REQUEST_MAPPER = new HashMap<>();

    private RequestMapping() {
    }
```

인스턴스를 만들지 않는 유틸 클래스에는 private 기본 생성자를 추가해주세요..!

> Utility classes, which are collections of static members, are not meant to be instantiated. Even abstract utility classes, which can be extended, should not have public constructors.
> 
> Java adds an implicit public constructor to every class which does not define at least one explicitly. Hence, at least one non-public constructor should be defined.

Reply:
빼먹고 추가를 못했었네요 감사합니다!